src/middlewared/middlewared/plugins/failover_/disabled_reasons.py

```diff
@@ -78,6 +79,9 @@ def get_local_reasons(self, app, ifaces, reasons):
         if self.middleware.call_sync('failover.config')['disabled']:
             reasons.add(DisabledReasonsEnum.NO_FAILOVER.name)
 
+        if self.middleware.call_sync('failover.reboot_required')['reboot_required']:
```

Comment:
Once you have statuses for both nodes, you'll want to do:

`reasons.add(DisabledReasonsEnum.LOC_FIPS_REBOOT_REQ.name)` and/or `reasons.add(DisabledReasonsEnum.REM_FIPS_REBOOT_REQ.name)`